osquery/dispatcher/scheduler.h

```diff
@@ -33,15 +35,19 @@ class SchedulerRunner : public InternalRunnable {
   /// The Dispatcher interrupt point.
   void stop() override {}
 
+  std::chrono::milliseconds getTimeDrift() const noexcept;
+
  protected:
   /// The UNIX domain socket path for the ExtensionManager.
   std::map<std::string, size_t> splay_;
 
   /// Interval in seconds between schedule steps.
-  size_t interval_;
+  std::chrono::milliseconds interval_;
```

Comment:
Interval is used on other places, did you check this doesn't cause any negative effects (e.g. due to casts)?

Reply:
yep, I did. There is no other place to use it.